exporter/fileexporter/README.md

```diff
@@ -1,35 +1,60 @@
 # File Exporter
 
 | Status                   |                       |
-| ------------------------ |-----------------------|
+| ------------------------ | --------------------- |
 | Stability                | [alpha]               |
 | Supported pipeline types | traces, metrics, logs |
 | Distributions            | [core], [contrib]     |
 
-This exporter will write pipeline data to a JSON file. The data is written in.
-[Protobuf JSON
-encoding](https://developers.google.com/protocol-buffers/docs/proto3#json)
-using [OpenTelemetry
-protocol](https://github.com/open-telemetry/opentelemetry-proto).
+Exporter supports the following features：
+
++ Support for writing pipeline data to a JSON file.
++ Support for rotation of telemetry files.
+
+The data is written in.[Protobuf JSON encoding](https://developers.google.com/protocol-buffers/docs/proto3#json). using [OpenTelemetry protocol](https://github.com/open-telemetry/opentelemetry-proto).
+
+
 
 Please note that there is no guarantee that exact field names will remain stable.
 This intended for primarily for debugging Collector without setting up backends.
 
 ## Getting Started
 
 The following settings are required:
-
 - `path` (no default): where to write information.
 
-Example:
+The following settings are optional:
+- `rotation` settings to rotate telemetry files.
+    - max_size:  [default: 100]: the maximum size in megabytes of the telemetry file before it is rotated.
```

Comment:
I think since we are stuck with v2 for now, we should call this `max_megabytes`. When v3 is available, we can add a new `max_size` field and deprecate this one.

Reply:
> Thanks for the quick updates. A couple more things. Apologies for marking up the docs so much. What you have is good but I'm being picky.

Your suggestions are very good, I can learn a lot from your suggestion. Thank you again~

Reply:
This one still needs `max_size` -> `max_megabytes`